.github/workflows/chromatic.ds-react-app-launchpad.yml

```
@@ -0,0 +1,11 @@
+name: "react-ds-app-launchpad/chromatic"
+on:
+  pull_request:
+    paths: [packages/react/ds-app-launchpad/**]
+jobs:
+  test-compile:
+    uses: ./.github/workflows/chromatic._template.yaml
```

Tested in #159 , the [job fails](https://github.com/canonical/ds25/actions/runs/13949753860). I think it's because there is a mismatch between name of the template file in the call and the file name itself - the extension is yml but you've used yaml here.

```suggestion
    uses: ./.github/workflows/chromatic._template.yml
```